.travis.yml

```diff
@@ -15,6 +15,8 @@ install:
   # dependencies via pip.
   - conda create --yes -n test-env -c biocore python=$PYTHON_VERSION scikit-bio jupyter click nose scikit-learn
   - source activate test-env
+  - conda install --yes -c bioconda blast=2.6.0
```

Comment:
Is this BLAST+ or legacy BLAST? We probably should use BLAST+ in QIIME 2, since legacy BLAST is pretty outdated at this point.